### PR TITLE
Ana/emoney sendmodal multiple denoms

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

In this PR I work on the `SendModal` component, making it posible to select from all the different tokens present in the balance and sending them.

I didn't test the "sending" bit. I will need to ask Henrik to send me over some testnet tokens. 

But for now you can check if you like how it is working right now. I had to do something maybe a bit weird that is calling the function to select the proper balance out of the `balances` array inside the `transactionData()` computed property. Perhaps this is not such a good practice. I commented on that part. 

Right now, the flow looks like this:

Selected token validation:

 

Tokens dropdown:

 

The corresponding balance is selected:

 

It remains the same for single-token networks:

 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
I am waiting for Charlie's flow. Once I have it, boom!

**Comment by Bitcoinera** [comment]
All suggested changes have been applied.

Still, please check if you get the first token in the list automatically selected when you open the SendModal. I want to make sure.

It should look like this just when opening (if connected to the `emoney-testnet`):

**Comment by faboweb** [comment]
Did you drop the idea of selecting the idea inline like Charlie sketched here: https://sketch.cloud/s/Oy9lm/a/DM8EZW ?

**Comment by Bitcoinera** [comment]
> Did you drop the idea of selecting the idea inline like Charlie sketched here: https://sketch.cloud/s/Oy9lm/a/DM8EZW ?

Well, I talked to Charlie and he had dropped it :woman_shrugging: 

I just follow his instructions in this matter :sweat_smile: 

Did you prefer it that other way?

**Comment by faboweb** [comment]
No, keep it like that when you agreed on that. Just wanted to point it out. :)
(but yes I liked the other version more 🙈)

File: src/ActionModal/components/SendModal.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@
         v-model.number="$v.address.$model"
         v-focus
         type="text"
-        placeholder="Address"
+        placeholder="Address..."
```
Comment (jbibla) [inline src/ActionModal/components/SendModal.vue]: ?

Comment (Bitcoinera) [reply src/ActionModal/components/SendModal.vue]: This is what Charlie's flow showed :laughing:

**Comment by Bitcoinera** [comment]
What is this really weird error showing up now in `testUnit`?

image

File: src/ActionModal/components/SendModal.vue (hunk 1 above)
Comment (jbibla) [reply src/ActionModal/components/SendModal.vue]: ah ok. sorry about that. let's remove it for now since it would be inconsistent with all our forms.

**Comment by faboweb** [comment]
@Bitcoinera did you run `jest` or `yarn test:unit`. the latter works for me

**Comment by faboweb** [comment]
adding `vue-jest` solved the issue 🤷‍♂

**Comment by faboweb** [comment]
changelog missing

File: src/ActionModal/components/SendModal.vue [hunk 2]
```diff
@@ -184,10 +215,24 @@ export default {
...
           this.address
         }`
       }
+    },
+    getDenoms() {
+      return this.denoms.map(denom => (denom = { key: denom, value: denom }))
+    }
+  },
+  watch: {
+    // we set the amount in the input to zero every time the user selects another token so they
```
Comment (jbibla) [inline src/ActionModal/components/SendModal.vue]: nice!

**Comment by jbibla** [comment]
> I liked the other version more

me too - but this is fine for now - let's merge!

**Comment by Bitcoinera** [comment]
> > I liked the other version more
> 
> me too - but this is fine for now - let's merge!

Yeah, me too actually. But easy to change

**Pull request merged**